azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -46,6 +46,10 @@ def apply_fake_pipeline_thread(fake_pipeline_thread):
     pass
 
 
+class FakeOperation(pipeline_ops_base.PipelineOperation):
+    pass
```

Comment:
Use the one defined in the conftest.py instead. The fixture is "op" (maybe should be renamed)